core/src/tile/tileManager.h

```diff
@@ -32,11 +32,7 @@ class TileManager {
 
 public:
 
-    /* Returns the single instance of the TileManager */
-    static std::unique_ptr<TileManager> GetInstance() {
```

Comment:
Ohh I think we needed a private static Tile manager instance for it to be truely singleton.